Gruntfile.js

```diff
@@ -248,57 +244,13 @@ module.exports = function (grunt) {
         }]
       }
     },
-
-    // Automatically inject Bower components into the app
-    wiredep: {
```

Comment:
I see you're removing wiredep. Do we need a replacement? Can you try `grunt serve:dist`?

Reply:
Well, something like Webpack would be the replacement for Wiredep. For now, we can just manually keep our vendor script imports in the Usemin block in app/index.html and they will still get concatenated into a vendor.js script in dist. The only thing we lose with Wiredep is the automatic population of the vendor script imports in app/index.html.